src/pages/ServiceDetails/ServiceInfo/__tests__/ServiceInfoPods.test.tsx

```diff
@@ -21,6 +21,8 @@ const pods: Pod[] = [
   }
 ];
 
+Date.prototype.toLocaleString = jest.fn(Date.prototype.toISOString);
```

Comment:
Good trick.
I would put a comment, just because I'm quite sure we will this one in the future :-).